scripts/participateInAuction.js

```diff
@@ -7,8 +11,16 @@ const { Gastimator } = require("./gasStation")
 const gasPriceUrl = require("../gas-config")
 const GasStation = new Gastimator()
 
+// const STATE_POOLING = 0 // Pooling
```

Comment:
Can these be removed? I also feel an enum would be the more appropriate way to represent these mutually exclusive phases.